README.md

```diff
@@ -143,53 +157,68 @@ fields:
 ### Debug configurations
 
 #### websockets
+
 This parameter allows you to turn off websockets when you want explicit polling every 5 seconds.
 The `interval` option from the Google Sheet configuration changes this value.
 
 example:
+
 ```yaml
 websockets: "false"
 ```
 
 ### Full examples
-Full configuration examples that incorporate either the query or fields method are present in 
+
+Full configuration examples that incorporate either the query or fields method are present in
 `config.query.example.yml` and `config.fields.example.yml` respectively.
 
 ## Rules (YARRRML)
 
-To convert and write back changes from the Google Sheet back to the resource, the synchronisation agent 
-uses the [RMLMapper](https://rml.io/). This mapper relies on declarative rules 
+To convert and write back changes from the Google Sheet back to the resource, the synchronisation agent
+uses the [RMLMapper](https://rml.io/). This mapper relies on declarative rules
 that define how the RDF data should be generated from the data on the Google Sheet.
 Write these rules in the form of [YARRRML](https://rml.io/yarrrml/) in the `rules.yml` file.
 
 You find an example in `rules.example.yml`.
 
 ## Start synchronisation app
+
 To set up and use the synchronisation agent, first make sure all the necessary dependencies have been installed by running
+
 ```shell
 npm i
 ```
 
 Afterwards, start the agent by running
+
 ```shell
 npm start
 ```
 
 ## Linting
-There is a linter set up for this project. To run the linter, execute the following command:
+
+ESLint linter set up for this project. To run the linter, execute the following command:
+
 ```shell
-npm run lint
+npm run lint:js
 ```
 
+There is also a markdown linter set up for this project. To run, execute the following command:
+`npm run lint:markdown` or `npm run lint:markdown:fix` to apply automatical fixes.
```

Comment:
This is not consistent with the code block of `npm run lint:js`. The latter is not inline, while this one is.